### ISSUE TITLE
Correct wiki page for importer

### ISSUE THREAD

**Issue by jasterba** [woocommerce/woocommerce#16800]
## Prerequisites

- [x ] I have searched for similar issues in both open and closed tickets and cannot find a duplicate
- [x ] The issue still exists against the latest `master` branch of WooCommerce on Github (this is **not** the same version as on WordPress.org!)
- [ x] I have attempted to find the simplest possible steps to reproduce the issue
- [ ] I have included a failing test as a pull request (Optional)

## Steps to reproduce the issue

1.Make CSV for import with custom atributes
2.Use importer
3.Wonder why it doesn't work

## Expected/actual behavior

When I follow those steps, I see... no custom attributes

I was expecting to see... custom attributes

## Isolating the problem

- [ x] This bug happens with only WooCommerce plugin active
- [ x] This bug happens with a default WordPress theme active, or [Storefront](https://woocommerce.com/storefront/)
- [x ] I can reproduce this bug consistently using the steps above

## WordPress Environment

<details>
```
Copy and paste the system status report from **WooCommerce > System Status** in WordPress admin here.
```
</details>

## TL;DR
Correct the wiki page. It says you need to have CSV Column Names: Attribute 1 Name, Attribute 1 Value(s), Attribute 1 Visible,... where it works only if there are no capitals in second parts: Attribute 1 name, Attribute 1 value(s), Attribute 1 visible,... 

Found this out thanks to exporter.

**Comment by mikejolley** [comment]
`woocommerce/includes/admin/importers/class-wc-product-csv-importer-controller.php` handles this. Let's see if we can make this work case-insensitive and ensure the mapping works correctly.
